script.js

```diff
@@ -1,11 +1,20 @@
 // Create an array called `colors` that contains three colors.
-
+var colors = ["red", "white", "blue"]{
+}
 // Log the last item in the array to the console.
-
+console.log
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
-
+colors.forEach(function(colors) {
+  console.log("My favorite color is []" + colors)
+});
+}
 // Create an `instructor` object that contains three key-value pairs.
+var= instructor ['height', 'class', 'period']{
 
+}
 // Add a `facial-hair` property to `instructor`.
+let instructor= {'facial-hair']
```

Comment:
to add a property that has punctuation that has a JS meaning (ie `-` in this case) we use bracket notation on the object and assign a value. So: `let instructor['facial-hair'] = "Santa Beard";`